_data/hosting_providers.json

```
@@ -2138,5 +2138,15 @@
     "plan": "",
     "reviewed": "",
     "note": ""
+  },
+  {
+    "name": "easyname",
+    "link": "https://www.easyname.com",
+    "category": "partial",
+    "tutorial": "https://www.easyname.com/en/support/hosting/202-how-can-i-activate-lets-encrypt-for-my-domain",
+    "announcement": "https://twitter.com/easyname_at/status/963002615717158913",
+    "plan": "https://www.easyname.com/en/hosting/free-lets-encrypt-ssl-certificate",
+    "reviewed": "",
```

Looking good, let's just add in a reviewed date!
```suggestion
    "reviewed": "2019.7.10",
```